tests/integration/fixtures/mini_sentry.py

```diff
@@ -462,5 +462,5 @@ def reraise_test_failures():
         "maxCustomMeasurements": 10,
     },
     "filters": {"version": 1, "filters": []},
-    "options": {},
+    "options": {"relay.span-usage-metric": True},
```

Comment:
Do you need it for all tests? If it's just for a single test, you can just enable it for that test.

Reply:
It should be enabled by default, will remove the option in https://github.com/getsentry/relay/pull/3259.